smallrye-llm-langchain4j-portable-extension/src/main/java/io/smallrye/llm/core/langchain4j/portableextension/LangChain4JAIServicePortableExtension.java

```diff
@@ -63,13 +61,8 @@ void processInjectionPoints(@Observes ProcessInjectionPoint<?, ?> event) {
     void afterBeanDiscovery(@Observes AfterBeanDiscovery afterBeanDiscovery, BeanManager beanManager)
             throws ClassNotFoundException {
         for (Class<?> aiServiceClass : detectedAIServicesDeclaredInterfaces) {
-            LOGGER.info("afterBeanDiscovery create synthetic :  " + aiServiceClass.getName());
-            final RegisterAIService annotation = aiServiceClass.getAnnotation(RegisterAIService.class);
-            afterBeanDiscovery.addBean()
-                    .types(aiServiceClass)
-                    .scope(annotation.scope())
-                    .name("registeredAIService-" + aiServiceClass.getName()) //Without this, the container won't create a CreationalContext
-                    .createWith(creationalContext -> CommonAIServiceCreator.create(CDI.current(), aiServiceClass));
+            LOGGER.info("afterBeanDiscovery create synthetic:  " + aiServiceClass.getName());
+            afterBeanDiscovery.addBean(new LangChain4JAIServiceBean<>(aiServiceClass, beanManager));
```

Comment:
Why change that part ? How is this rellted to Fault Tolerance ?

Reply:
The `InterceptorFactory` registers the proxy bean to the `FaultInterceptorBinding`, used register the AI Service for Fault Tolerance.

Reply:
ok